ui/tests/helpers/components/mount-backend-form-selectors.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+export const MOUNT_BACKEND_FORM = {
+  header: '[data-test-mount-form-header]',
+  mountType: (name: string) => `[data-test-mount-type="${name}"]`,
```

Comment:
Since it seems unnecessary to have a whole file for one selector, I'm fine with this coming from the secret-engine-selectors (though I see you removed it from there). We could rename the secret engine selector file "engine-mount-selectors" if we want it to be more general

Reply:
Another idea is just defining this in the file it's used the most (the component test) and then elsewhere just using `data-test...`

Reply:
I had similar thoughts. I like the idea of moving back to secret engine selectors and renaming that file. long term that better captures the situation.

Reply:
After some moving and shuffling, i came back to as is. I added a comment in the file why. Not ideal, but I think until we can decouple secret engines and auth methods from the start of the flow, we've got some overlap and keeping this scoped to the component helper is the most clear, IMO.